### PR TITLE
Add 'Why Firefox' SEO page. (Fixes #6815)

### LINKED ISSUE #6815: Why Firefox - SEO Page
## Description
SEO Page to drive SERP ranking entitled ""Why Firefox could be the best browser for you" 

### Estimates

| Estimates |  |
| ------------- | ------------- |
| TL | |
| PM | 2 hours |
| ENG | |
| SEO | ? |
| ANA | |
| DES | |
| COPY | |

---

[Copy(https://docs.google.com/document/d/151gPy7rcGvd8Y9PCbtvFfFyLv066zi7F27DnGF5euYo/edit)

#### :yellow_heart: Success Criteria :yellow_heart:
- [x] Is approved by @rraue and Winston (page on demo server)
- [ ] Deployed at the URL specified by @rraue  - https://www.mozilla.org/firefox/best-browser/
- [ ] Strings are extracted for L10n
- [ ] Anika and Malcolm are notified when strings are available in Pontoon (@ejregithub)

#### Tasks
- [ ] Code the page using this file in [Abstract:](https://share.goabstract.com/3c00a324-3de4-4936-9997-6299a6133df8) noting that the layer the design lives on:
Revised Design layer > Choosing Browser Landing Page

### PR DESCRIPTION
**Pull request by amychurchwell**
## Description
Adds an SEO page at firefox/best-browser/

## Issue / Bugzilla link
#6815

## Testing
- Copy reflects the [copy doc](https://docs.google.com/document/d/151gPy7rcGvd8Y9PCbtvFfFyLv066zi7F27DnGF5euYo/edit)

https://bedrock-demo-achurchwell.oregon-b.moz.works/firefox/best-browser/

### CODE REVIEW

**Comment by amychurchwell** [comment]
Adding "Do Not Merge" label because I would like to create a pull request to add styles to blockquote `<cite>`'s in protocol and avoid writing custom styles here.

File: bedrock/firefox/templates/firefox/best-browser.html [hunk 1]
```diff
@@ -0,0 +1,162 @@
...
+       </p>
+       <p>
+         {% trans monitor='https://monitor.firefox.com/' %}
+         Firefox is offering something new to keep you safe: <a href="{{ monitor }}">Firefox Monitor</a>. It’s a free service that will alert you if there are any public hacks on your accounts and let you know if your accounts got hacked in the past. Another neat feature is the Green Lock. It looks like a small green icon at the top left side of the browser window. If you’re on Firefox and see the green lock, it means the website is encrypted and secure. If the lock is grey, you might want to think twice about entering any sensitive information.
+         {% endtrans %}
+       </p>
+       <blockquote aria-hidden="true" role="presentation">
+        <p>
+          {% trans %}
+          We visit hundreds or even thousands of websites each day, and you can't expect users to make security and privacy decisions for each of these sites. That is why a browser that gives you more control is so important - because it offers real, meaningful protection.
```
[the remark this answers is not in the document]
Comment (amychurchwell) [reply bedrock/firefox/templates/firefox/best-browser.html]: thank you! good catch. 🙂

File: media/css/firefox/bestbrowser.scss [hunk 2]
```diff
@@ -0,0 +1,71 @@
...
+    background: $color-purple-90 url('/media/img/firefox/best-browser/hero-pattern.png') 55% 60% no-repeat;
+    @include background-size(2500px);
+    color: #fff;
+    text-align: center;
+    min-height: 520px;
+    margin-bottom: $spacing-xl;
+
+    .hero-content {
+        padding-top: $layout-xl;
+        padding-bottom: $layout-xl;
```
[the remark this answers is not in the document]
Comment (craigcook) [reply media/css/firefox/bestbrowser.scss]: Sorry, my previous comment was rambly and confusing...

You added the `mzp-l-content` class which includes left and right padding, but then you also shortened this to `padding: $layout-xl 0;` which in turn overrides that left and right padding.

You can revert this back to the separate declarations for top and bottom so the side padding will come from the layout class.

[the remark this answers is not in the document]
Comment (amychurchwell) [reply media/css/firefox/bestbrowser.scss]: @craigcook Thank you for the clarification! I guess I was confused, haha. I pushed the fix.

**Pull request merged**